events/scheme_events/wc_plague_scheme/wc_plague_ongoing_events.txt

```diff
@@ -0,0 +1,890 @@
+﻿#Events for Plague Scheme
+
+namespace = plague_ongoing
+
+############################################
+#  ONGOING OPPORTUNITY EVENTS 3000 - 3999
+############################################
+
+#3001 - 3199 are events mainly triggered by agents
+#3201 - 3299 are events mainly triggered by target
+#3301 - 3399 are events mainly triggered by owner
+#3401 - 3499 are events exclusive to children
+
+
+#######################
+# Drunkard Agent
+# by Linnéa Thimrén and Petter Vilberg
+#######################
+plague_ongoing.3010 = {
```

Comment:
The text is almost the same as vanilla event.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/133298131-a4fadc62-4f7d-41ef-9354-d60c0623d969.png)

</details>

That's what I was [talking about](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/647#discussion_r702279914). You don't need to copy a vanilla event if you just change one localization line.